app/src/nxt/res/layout/fragment_nxt_firmware.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I ran it on a Nexus 5X API 22 (Android 5.1.1) emulator. I think I didn't change its appearance but other settings have such sub-windows. Is this device-dependence problem? I'm not sure... 😅  
Do you think we should expand the width of this sub-screen?